README.md

```diff
@@ -1 +1,62 @@
-# activemq-artemis-console
+# Apache ActiveMQ Artemis Console
+
+
+The [Apache ActiveMQ Artemis](https://activemq.apache.org/components/artemis/) Console Plugin is written using [Hawtio v4](https://github.com/hawtio/hawtio).
+The plugin is written in TypeScript. Since a Hawtio plugin is based on React and [Webpack Module Federation](https://module-federation.github.io/),
+this project uses Yarn v3 and [CRACO](https://craco.js.org/) as the build tools.
+
+The WAR file created by this project is consumed by ActiveMQ Artemis but can be developed and run standalone.
+
+
+### Build
+
+The following command first builds the `artemis-console-extension` frontend project and then compiles and packages 
+the main Java project Web Archive in `artemis-console-war`.
+
+```console
+mvn clean install
+```
+
+Building the frontend project 'artemis-console-extension' can take time, so if you build it once and make no changes on the project afterwards, you 
+can speed up the whole build by skipping the frontend part next time.
+
+```console
+mvn install -Dskip.yarn
+```
+
+### Test run
+
+You can quickly run and test the console by using `jetty-maven-plugin` configured in `pom.xml`. It launches an embedded 
+Jetty server and deploys the plugin WAR application. From the 'artemis-console-war' directory run:
+
+```console
+cd artemis-console-war
+mvn jetty:run -Dskip.yarn
+```
+
+You can access the Artemis console with the sample plugin at: <http://localhost:8080/console/>
```

Comment:
When I try this, I see Jetty starting, and can load a Hawtio console by visiting the link, but there is nothing about Artemis in it...there are Connect, JMX, and Runtime tabs in the left menu. After a bit of blind prodding and looking at my broker console I figured out I needed to 'Connect' it to 'http://localhost:8161/console/jolokia" ...maybe need some more doc here? Such as referencing the _later_ doc that I only then noticed is at the bottom of the readme for the _other_ way of doing a local startup for dev (which I haven't tried as I don't have yarn system installed, and didn't try to figure out how to use the ones from the maven build)

Reply:
Ive added some docs explaining how to use.